delivery_state/models/res_config_settings.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    send_delivery_confirmation = fields.Boolean(
+        string="Send Delivery Confirmation",
```

Comment:
Put this as string in the company field, and don't put anything here and the related "magic" will happen